### ISSUE TITLE
response by email posted whole email

### ISSUE THREAD

**Issue by steviepubliclab** [publiclab/plots2#6990]
Hello, 
I believe the person (gloryaec) who posted on this question used the respond by email feature. It looks like it carried over the footer of the email as well:

https://publiclab.org/questions/stevie/12-16-2019/are-there-simple-tests-for-nitrates-in-well-water

**Comment by jywarren** [comment]
Screenshot here! Stevie is right!

![image](https://user-images.githubusercontent.com/24359/71021023-76b14380-20cb-11ea-8c2c-35ace7c3ad86.png)

We have a whole system to detect quoted text from emails; it's here:

https://github.com/publiclab/plots2/blob/a0aaa05476c7c01138c8d1440689eba6614f2bc3/app/models/comment.rb#L449-L463

However it looks like it failed on this message. I recovered the email and we should test it against the filter in `comment.rb`:

https://gist.github.com/jywarren/16f976c8bef0b1d22ba44484c8758e25

We could test it manually in https://rubular.com/ and figure out what went wrong, and adjust the filter?

**Comment by SidharthBansal** [comment]
Published

**Comment by steviepubliclab** [comment]
Hi great! I just tried to reply by email to test this out. I was replying to a comment on a research note and it didn't go thorough. Perhaps this is a different issue. This is what it looked like on my email: 

<img width="988" alt="Screen Shot 2020-01-21 at 1 32 40 PM" src="https://user-images.githubusercontent.com/7490478/72832273-8c79d180-3c52-11ea-90c7-475be5c467e1.png">

It didn't show up on this research note: https://publiclab.org/notes/nshapiro/09-26-2016/build-a-plant-based-air-purifier#comments

**Comment by jywarren** [comment]
I think this may have been published on the GCI task dashboard but not yet taken up or solved. Perhaps @Uzay-G or @VladimirMikulic may be interested in it, if there is any time left, or we can boost it another way!

**Comment by jywarren** [comment]
Ah and yes, I think you have ID'ed another issue about it not posting at all, and will post that. Can you forward me both emails so I can collect all relevant info just in case? Thanks!

**Comment by jywarren** [comment]
Posted it here! https://github.com/publiclab/plots2/issues/7321 Thank you!!!

**Comment by Uzay-G** [comment]
> I think this may have been published on the GCI task dashboard but not yet taken up or solved. Perhaps @Uzay-G or @VladimirMikulic may be interested in it, if there is any time left, or we can boost it another way!

I will investigate to try and debug the error :+1:

**Comment by Uzay-G** [comment]
Testing the comment filter on the gist you provided fails.

**Comment by jywarren** [comment]
Are we able to add a new filter step that scrubs the content in our own comment emails? Perhaps, looking for standard text from one or all of these templates? https://github.com/publiclab/plots2/tree/master/app/views/comment_mailer

**Comment by Uzay-G** [comment]
Yeah that could work. But maybe could you explain what the `.+` in the regex does? I understand the grouping of the results but the use of  `.+`  confuses me. Thanks!
